frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -151,7 +157,7 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
     return;
```

Comment:
still applies: https://github.com/openshift/console/pull/5977#discussion_r459952140

Reply:
also this whole thing should not get fired when isProviderImport

Reply:
done